src/app/pages/storage/modules/pool-manager/utils/generate-vdevs/generate-vdevs-maximize-dispersal.spec.ts

```diff
@@ -130,11 +130,11 @@ describe('GenerateVdevsService - maximize dispersal', () => {
       ]),
       [VdevType.Log]: expectDisks([
         ['small-ssd1'],
-        ['small-ssd2'],
+        ['enclosure2-never-used-too-small'],
```

Comment:
1. `enclosure2-never-used-too-small` was designed not to be used in tests. Let's change it to 1GB, so that it doesn't appear. Not sure why it worked before.
2. Even after I change it to 1Gb, it tests still pass, which doesn't appear to be correct.

Reply:
if I try with 0.99GB it's not appearing